pkg/pub_worker/lib/src/analyze.dart

```diff
@@ -152,12 +152,28 @@ Future<void> _analyzePackage(
       await pana.stdin.close();
 
       await Future.wait<void>([
-        pana.stderr.forEach(log.add),
-        pana.stdout.forEach(log.add),
+        pana.stderr
+            .transform(utf8.decoder)
+            .transform(LineSplitter())
```

Comment:
What kind of things can this throw?

Certainly it can throw FormatException, do we want it to be UTF8? If so I certainly think we should accept malformed utf8 from the subprocess.

Reply:
How about `Utf8Decoder(allowMalformed: true)`? I think unconditional stream merge is similarly error-prone.